esy-solve/Resolver.re

```diff
@@ -488,7 +488,15 @@ let package =
   open RunAsync.Syntax;
   let key = (resolution.name, resolution.resolution);
 
-  let ofVersion = (version: Version.t) =>
+  let ofVersion = (~override=?, version: Version.t) => {
+    let patch_overrides = (pkg: InstallManifest.t) =>
+      switch (override) {
+      | Some(override) => {
+          ...pkg,
+          overrides: Overrides.add(override, pkg.overrides),
+        }
+      | None => pkg
+      };
```

Comment:
Another nit, can you move the `patch_overrides` function outside which accepts `override` as an optional parameter?

Overall the changes look good

Reply:
Will do that, I was also thinking on moving this to the `InstallManifest` module so that we do something like `InstallManifest.with_override`

Reply:
@ManasJayanth yup

Reply:
Sorry @melwyn95 this one is a technical debt fgor the future